src/presentation-markup.html

```diff
@@ -6315,12 +6295,10 @@ <h5>Attributes</h5>
    </section>
  
    <section class="fold">
-    <h5>Examples</h5>
+    <h5 id="presm_munderover_ex">Examples</h5>
  
-    <p>The MathML representation for the example shown above with the first
-    expression made using separate <code class="element">munder</code> and
-    <code class="element">mover</code> elements, and the second one using an
-    <code class="element">munderover</code> element, is:</p>
+    <p>This example shows the subtle difference between nesting <code class="element">munder</code> inside
```

Comment:
I think it's too subtle to highlight (I'm not sure a difference is mandated at all in that case is it? I'd use a non subtle example, where it uses subscript layout

> This example shows the difference between nesting <code class="element">munder</code> inside
      <code class="element">mover</code> and using <code class="element">munderover</code>is particularly apparent when not in `displaystyle`, when the limits may be placed as for <code class="element">msubsup</code>`

```
<mstyle displaystyle="false">
 <mover>
 <munder>
  <mo>∑</mo>
  <mi>x</mi>
 </munder>
 <mi>n</mi>
 </mover>
 <mo>+</mo>
 <munderover>
  <mo>∑</mo>
  <mi>x</mi>
  <mi>n</mi>
 </munderover>
</mstyle>
```

Reply:
or probably with a more mathematically meaningful example, but the above does show the layout difference, firefox shows

![image](https://user-images.githubusercontent.com/1268738/167366644-fc1c789d-5a88-4d82-9019-879e3656e359.png)

Reply:
Good idea to show an example with moveable limits for this. I have changed the example to use `i` instead of `x` since `n` is usually an integer.